assets/src/edit-story/components/panels/layer/test/layer.js

```diff
@@ -88,6 +88,6 @@ describe('Layer', () => {
         failureThreshold: 0.01,
         failureThresholdType: 'percent',
       });
-    });
+    }, 10000);
```

Comment:
Hmm. That's a lot of timeout. Why would it be this slow?

Reply:
The default is 5s (5000ms), this doubles it to 10s (10000ms).

Especially on CI this test might take longer than 5s overall since it starts Puppeteer and needs to take a screenshot and compare it.